app/controllers/bars_controller.rb

```diff
@@ -22,6 +22,7 @@ def create
 
   def show
     @bar = Bar.find(params[:id])
+    @reviews = @bar.reviews.order(created_at: :desc)
```

Comment:
I would consider putting the `order(created_at: :desc)` in the view rather than the controller as the order the list is being displayed in is more of a view consideration than a controller job.